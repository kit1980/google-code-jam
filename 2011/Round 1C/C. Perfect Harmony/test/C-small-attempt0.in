40
3 2 100
3 5 7
4 8 16
1 20 5 2
13 1 10000
6030 3924 1855 313 8316 9005 3365 8660 2034 9251 6888 9783 6130
10 8281 9937
552 69 828 276 414 184 207 72 92 138
5 3004 8588
7472 6020 6115 338 6034
10 61 119
2 7920 4680 3 5040 20 12 9720 2520 30
51 868 1733
867 1734 1734 1734 1734 867 1734 867 1734 1734 867 867 1734 867 1734 867 1734 867 867 1734 1734 867 867 1734 867 1734 1734 867 1734 1734 867 1734 867 1734 1734 1734 867 867 867 1734 867 867 1734 867 867 1734 1734 1734 867 867 867
5 5672 9138
8562 2448 8785 4090 5498
10 61 119
4320 40 9360 4 1440 20 60 10 7920 5040
5 6854 8913
9746 1135 1680 3328 3300
100 11 30
5190 540 7860 8160 1740 9330 7980 630 3360 8010 7500 5430 4170 4560 7800 4830 9510 5790 1200 5760 4620 9900 4140 3660 3930 4680 2820 4950 1140 780 2250 5850 5880 6480 4590 5490 990 7650 2550 5580 5280 600 3420 8220 9840 1110 420 9180 7170 8070 4200 8850 2790 6660 6540 690 9540 8940 7320 1290 6150 7140 4800 330 4410 2670 1320 6720 7050 6780 8880 9570 6840 7440 8760 6210 3030 1920 2940 7920 4050 300 6450 1260 9690 9390 2610 9600 1230 6810 1770 9930 2850 9720 5610 9120 1800 1170 5250 9870
5 2820 8253
5570 6334 2461 5284 5317
5 7196 7307
7190 8535 8534 8639 286
6 2 10000
9973 9967 9931 9949 9929 9941
5 4022 4573
1528 8871 3071 3646 1519
4 2 10000
4 49 17 3
5 1758 2434
7383 4500 3637 5944 2814
100 1 10000
2682 1125 8441 5250 2372 5629 441 2349 675 4681 194 4312 8276 6602 5841 3248 7478 5471 2398 7659 6733 7235 4323 7341 4743 2045 9475 9857 9653 5366 2715 1021 5364 4934 8028 4596 4827 4112 1184 3773 1816 4848 853 9212 5141 6071 8461 1059 2416 8586 150 1470 7328 6188 7704 7650 4534 9398 2838 5392 524 1683 8636 5967 382 8776 9447 3723 4189 7600 7758 1789 5670 5817 8949 8941 9911 124 5477 5424 4126 7636 5367 7449 9243 2515 6977 1585 333 2112 6008 8305 7594 6064 8425 8112 1894 8009 1146 4067
5 6922 8985
5008 5588 3665 9446 3352
5 3120 6394
1331 7675 8655 3350 3638
100 2 10000
1085 1178 2604 1953 589 558 2883 2294 1364 2201 775 2046 1488 837 2170 1023 2387 2139 124 806 1705 1860 2666 682 2356 62 868 3131 1302 403 2418 1550 1674 372 1891 1519 1209 1240 1054 2728 217 527 93 341 651 3100 1147 496 2914 2976 1736 1922 930 713 2945 248 279 1984 2511 1395 310 3007 1767 1116 992 1612 744 3069 2635 2449 1333 1829 155 1643 1457 3038 620 1271 2759 2325 1426 2573 2852 2108 186 2790 465 961 1581 2697 2542 2480 2263 2821 2015 434 2077 1798 2232 899
3 2 50
3 5 75
10 2 10000
11 15 10 2 30 1 7 5 3 6
10 2 10000
11 3 15 6 1 10 5 2 2310 7
51 616 1230
123 123 3690 3690 3690 3690 3690 123 123 123 3690 3690 3690 123 123 123 3690 3690 123 3690 3690 123 3690 123 3690 3690 123 123 123 123 123 123 3690 123 3690 3690 3690 3690 123 123 3690 123 3690 123 123 123 3690 3690 123 3690 3690
5 7239 7456
8131 8996 6586 4245 3514
7 5 10000
9714 9942 9762 9822 9978 9678 9726
5 4202 9280
9899 3379 3140 9006 3853
11 2 10000
1 1 8633 1 1 8633 8633 1 1 8633 8633
5 1989 8836
6424 1603 8188 4195 9134
5 6679 8027
9277 1881 9489 6118 161
5 8812 9352
4224 3728 2505 825 9583
5 2029 7801
1010 4051 3917 2533 2108
5 8604 9976
5399 9847 3975 4715 5283
50 2 10000
36 21 504 1260 168 12 18 24 112 16 7 10 40 1008 3 2520 336 126 4 9 1 30 5040 20 140 144 14 280 2 60 420 8 56 630 1680 720 210 15 72 105 315 840 252 45 560 120 35 63 360 90
5 2850 6012
5462 8038 6712 9083 9100
5 5312 7964
7831 3379 8757 9872 9405
5 2012 4087
5063 5649 71 6579 9201
2 5001 10000
5000 2500
5 4734 8813
1126 5014 6543 1161 6081
