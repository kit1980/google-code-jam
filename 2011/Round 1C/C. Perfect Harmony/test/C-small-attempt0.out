Case #1: NO
Case #2: 10
Case #3: 1
Case #4: 9936
Case #5: NO
Case #6: NO
Case #7: NO
Case #8: NO
Case #9: NO
Case #10: NO
Case #11: 15
Case #12: NO
Case #13: NO
Case #14: NO
Case #15: NO
Case #16: 9996
Case #17: NO
Case #18: 1
Case #19: NO
Case #20: NO
Case #21: 31
Case #22: 15
Case #23: 2310
Case #24: 2310
Case #25: 738
Case #26: NO
Case #27: 6
Case #28: NO
Case #29: 89
Case #30: NO
Case #31: NO
Case #32: NO
Case #33: NO
Case #34: NO
Case #35: 5040
Case #36: NO
Case #37: NO
Case #38: NO
Case #39: 10000
Case #40: NO
